Taskfile.yml

```diff
@@ -0,0 +1,12 @@
+# https://taskfile.dev/
+
+version: '3'
+
+tasks:
+  default:
+    cmds:
+      - NXF_VER=21.04.0-EDGE nextflow run main.nf -params-file params/local.yml -resume -with-tower
```

Comment:
Let's refactor to add the latest stable release :) 

You can check on the main NF repository.

Reply:
Ok, I'll refactor using 21.04.0, last stable release without edge.